packages/client/src/services/notification-service/notification.service.ts

```diff
@@ -0,0 +1,34 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { SwPush } from '@angular/service-worker';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class NotificationService {
+    private readonly VAPID_PUBLIC_KEY =
+        'BC7J1hDqhmWAb-N88JCvA_vgasQ433M8sD4Fl6qWyfiinOgW3qh62vJfrs2wWLg6I26JdsXwoXX2u7YS13XQbJA';
```

Comment:
Same thing here, maybe put them in the environment file (`packages/client/src/environments`)

Reply:
I will put then in a .env file